### DISCUSSION TITLE
Ability to override orientation preference

### DISCUSSION THREAD

**Discussion by kretes** [nroduit/Weasis#503] [General]
Hello,

Is there a way in weasis to read & present the data in a specific orientation?
I have some data that when read is presented in MPR using 'R' 'P' and 'F' directions.
However I would like the data to be always presented using 'R' 'A' and 'H' directions.
I can rotate each view to achieve that, and then - 'R' 'A' and 'H' directions appear, but I would like it to be done automatically by setting a preference. Is there such a thing? I didn't find an option for that.

Thanks!

**Comment by nroduit** [comment]
Do you mean the feet are oriented upwards? With the 3D viewer, is the orientation correct?

Is it working correctly with other viewers or is it related to the dataset?

**Reply by kretes** [reply]
Yes, I mean that 'F' is on the top in 3D and indeed what is shown in the image has feet on the top..
I'm looking at it more from technical perspective, rather than medical.
Tool like Slicer3D displays the same dicoms normally. 
It seems to me that it forces always the same anatomical orientation when reading data.

**Comment by nroduit** [comment]
I've never seen this kind of issue.  We use the DICOM 3D vectors for the anatomical orientation. Please provide a sample or if you cannot at least screenshots after opening view 2D, MPR and view 3D.
